server/src/auth/auth.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Inject, Injectable } from '@nestjs/common';
+import { JwtPayload } from './interfaces/jwt-payload.interface';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class AuthService {
+    constructor(
+        @Inject('JwtService') private readonly jwtService: JwtService,
+    ) {}
+
+    createToken(email: string) {
+        const user: JwtPayload = { email };
```

Comment:
I think user: payload is awkward.
payload: Payload or
user:User isnt better?

Reply:
There is no email field in our user object.

I think address is needed

Reply:
@inDlife 
I think `email` is a variable which is parameter.

Reply:
I recommend an idea `const userPayload: UserPayload` and `interface UserPayload`.